recipe/test_jaxlib.py

```diff
@@ -8,4 +8,10 @@
 y = xops.Sin(x)
 computation = c.Build()
 cpu_backend = xc.make_cpu_client()
-compiled_computation = cpu_backend.compile(computation)
+
```

Comment:
conda-forge has this https://github.com/conda-forge/jaxlib-feedstock/blob/main/recipe/test_jaxlib.py